CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ file. This file is structured according to http://keepachangelog.com/
 ### Added
 - `id` in `case` for the source case id
 - `contact.preferredLanguages` as array in `case`
+- Replace human readable `id` on `person.contact.sourceCases` by foreign key `personId`
```

Comment:
It's not replaced, it's wo different concepts.

- `id` was the source case id, which is an arbitrary id assigned by some external health agency
- `personId` is the reference to the document where the person information to this case is saved

I think this should be properly reflected in the CHANGELOG.